cypress/integration/core-opensearch-dashboards/opensearch-dashboards/workspace-plugin/mds_workspace_association.spec.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+const miscUtils = new MiscUtils(cy);
+const workspaceName = 'test_workspace_collaborators';
+let workspaceId;
+let dataSourceTitle1 = 'no_auth_data_source_title_1';
+let dataSourceTitle2 = 'no_auth_data_source_title_2';
+let dataSourceId1;
+let dataSourceId2;
+if (
+  Cypress.env('WORKSPACE_ENABLED') &&
+  Cypress.env('SECURITY_ENABLED') &&
```

Comment:
Do we need security enabled for testing data source association?

Reply:
Do you mean we don't need this verification?